src/scram.erl

```diff
@@ -30,60 +30,64 @@
 
 %% External exports
 %% ejabberd doesn't implement SASLPREP, so we use the similar RESOURCEPREP instead
```

Comment:
Is the comment still valid?

Reply:
Honestly I suppose not anymore but I'm not even sure what was meant 🤔

Reply:
@NelsonVides SASLPREP and RESOURCEPREP are the profiles describing how to normalise Unicode strings before processing them further. In short, these say: convert to lowercase, do not allow some characters and map them to placeholders / equivalents, etc. See https://tools.ietf.org/html/rfc7622 if you're interested, but there's not a lot of action there.

Usually, this is handled by calling `stringprep`, but can be done manually, too.